### PR TITLE
Ticket info data model

### PR DESCRIPTION
**Pull request by DAloG**
Data model class that able to parse raw string from QR Code.

### CODE REVIEW

File: iOS/UZTickets/src/core/UZTTicketInfo.h [hunk 1]
```diff
@@ -0,0 +1,102 @@
+//
+//  UZTTicketInfo.h
+//  UZTickets
+//
+//  Created by Алексей Демедецкий on 01.06.15.
+//  Copyright (c) 2015 DAloG ltd. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
```
Comment (xNekOIx) [inline iOS/UZTickets/src/core/UZTTicketInfo.h]: @import

File: iOS/UZTickets/tests/UZTTicketInfoTests.m [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+@"АЛЕКСЕЙ ДЕМЕДЕЦКИЙ\n"
+@"MПC\n"
+@"\n"
+@"0.00\n"
+@"\n"
+@"\n"
+@"000B37D3-61738C35-0001\n"
+@"\n"
+@"50182\n"
+@"CBCE3F97619DA86CF45297C1D8ABB3ED75D1A0CD\n";
```
Comment (xNekOIx) [inline iOS/UZTickets/tests/UZTTicketInfoTests.m]: It is possible to do like this? :cool: :sunglasses:

File: iOS/UZTickets/tests/UZTTicketInfoTests.m [hunk 3]
```diff
@@ -0,0 +1,64 @@
+//
+//  UZTTicketInfoTests.m
+//  UZTickets
+//
+//  Created by Алексей Демедецкий on 01.06.15.
+//  Copyright (c) 2015 DAloG ltd. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
```
Comment (xNekOIx) [inline iOS/UZTickets/tests/UZTTicketInfoTests.m]: @import

File: iOS/UZTickets/src/core/UZTTicketInfo.h (hunk 1 above)
Comment (DAloG) [inline iOS/UZTickets/src/core/UZTTicketInfo.h]: This is auto generated code. Do you know how to turn it on on xcode level?

Comment (xNekOIx) [inline iOS/UZTickets/src/core/UZTTicketInfo.h]: nope. Hope they'll fix it some day.

**Comment by xNekOIx** [comment]
Fix imports and it is good to go

File: iOS/UZTickets/src/core/UZTTicketInfo.h (hunk 1 above)
Comment (DAloG) [inline iOS/UZTickets/src/core/UZTTicketInfo.h]: You don't actually need to use the @import keyword. If you opt-in to using modules, all #import and #include directives are mapped to use @import automatically. That means that you don't have to change your source code (or the source code of libraries that you download from elsewhere). Supposedly using modules improves the build performance too, especially if you haven't been using PCHs well or if your project has many small source files.

> http://stackoverflow.com/questions/18947516/import-vs-import-ios-7

**Comment by DAloG** [comment]
I don't want rewrite generated code. Especially when there is no reason to do that.

**Pull request merged**